### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by JosephinaBos**
There are three commits in here: two that change a couple of titles in summary.md (and removes the weird line in website.css) and one that adds extra info to the technology chapter

### CODE REVIEW

File: introduction/the-technology.md [hunk 1]
```diff
@@ -13,14 +13,14 @@ Unlike co-browsing, screen-sharing does not limit itself to the web browser. Wit
 
 Also, screen-sharing solutions are pixel-based, which means that the controller constantly takes snapshots of the screen and then sends these to the other side in compressed form. The drawback of this approach is that screen updates are slow and of low quality.
 
-What most screen-sharing and co-browsing solutions, other than Surfly, have in common, is that both rely on external software that needs to be installed by both users. This makes it unsuitable for most web situations as people are often unwilling to install extra software that circumvents the browser's security measures.
+What most screen-sharing and co-browsing solutions, other than Surfly have in common, is that both rely on external software that needs to be installed by both users. This makes it unsuitable for most web situations as people are often unwilling to install extra software that circumvents the browser's security measures.
```
Comment (jsnjack) [inline introduction/the-technology.md]: I think that coma between Surfly and have is still needed here :/

File: introduction/the-technology.md [hunk 2]
```diff
@@ -13,14 +13,14 @@ Unlike co-browsing, screen-sharing does not limit itself to the web browser. Wit
...
 
-There are also many limitations to this approach. For example, since the iframes on the original page come from an external Javascript source, they can't be controlled by the Javascript that the co-browsing solution sends over to the users. Furthermore, audio and video are usually not synchronized. But more importantly, it is unsafe to handle logged-in sessions, as they are usually only possible if login credentials are sent to the followers; otherwise the followers do not have access to session-specific data.
+There are also many limitations to this approach. One major issue is that the iframes on the original page come from an external Javascript source. This means that, because of the cross domain policies, these iframes can't be controlled by the Javascript that the co-browsing solution sends over to the users. Second, audio and video are usually not synchronized. Furthermore, it is unsafe to handle logged-in sessions, as they are usually only possible if login credentials are sent to the followers; otherwise the followers do not have access to session-specific data.
```
Comment (muodov) [inline introduction/the-technology.md]: `external Javascript source` is confusing, there is no such term. I would say "the iframes loaded from another domain have a different Javascript scope".

**Comment by muodov** [comment]
Just one small remark, otherwise looks good

File: introduction/the-technology.md (hunk 1 above)
Comment (JosephinaBos) [reply introduction/the-technology.md]: Okidoki, thanks for the review!

File: introduction/the-technology.md (hunk 2 above)
Comment (JosephinaBos) [reply introduction/the-technology.md]: Ok sounds good, thanks!

**Pull request merged**